### PR TITLE
4.11 RN for TELCODOCS-339 - Recovery of clusters after a failed upgrade

### PR DESCRIPTION
**Pull request by slovern**
Version(s):4.11

Issue: https://issues.redhat.com/browse/TELCODOCS-339

"**Recovery of clusters after a failed update**"  (Last entry in _Scalability and performance_ section)
Link to docs preview: 
http://file.emea.redhat.com/slovern/RN-4.11-[TELCODOCS-339](https://issues.redhat.com//browse/TELCODOCS-339)/release_notes/ocp-4-11-release-notes.html#ocp-4-11-scalability-and-performance

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -1050,6 +1050,11 @@ The Special Resource Operator (SRO) includes a consistent log output format with
 
 Before this update, SRO did not support connecting to registries in disconnected environments. This release provides support for disconnected environments for which the driver container is hosted in a registry outside of the {product-title} cluster.
 
+[id="ocp-4-11-talm-upgrade-recovery"]
+====  Recovery of clusters after a failed upgrade
```
Comment (opayne1) [inline release_notes/ocp-4-11-release-notes.adoc:1054]: Update is the preferred usage see  https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update
```suggestion
====  Recovery of clusters after a failed update
```

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -1050,6 +1050,11 @@ The Special Resource Operator (SRO) includes a consistent log output format with
...
+
+For {sno}, you can use the {cgu-operator-first} operator to create a backup of a current deployment before an {product-title} version upgrade. If the upgrade fails, you can use the backup to return the cluster to its pre-upgrade state.
```
Comment (opayne1) [inline release_notes/ocp-4-11-release-notes.adoc:1056]: Operator should be capitalized. See Operator in "O" section of SSG https://redhat-documentation.github.io/supplementary-style-guide/#_o

Comment (opayne1) [reply release_notes/ocp-4-11-release-notes.adoc:1056]: See above comment about upgrade/update.

File: release_notes/ocp-4-11-release-notes.adoc (hunk 1 above)
Comment (opayne1) [reply release_notes/ocp-4-11-release-notes.adoc:1054]: I would also recommend revising this subheading for present tense.

**Review by opayne1 — COMMENTED** [review]
Just a few style guide nits.

NOTE: This is a review of content _ONLY_. PR as is in the GA rel notes section should not be merged.

**Comment by slovern** [comment]
Merging in con PR for 4.11 RN notes for RAN: https://github.com/openshift/openshift-docs/pull/50930

**Pull request closed without merging**